Containment/forensics/README.md

```diff
@@ -61,7 +65,7 @@ Since the script uploads content to an s3 bucket, the instance on which the scri
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Sid": "VisualEditor0",
+            "Sid": "VisualEditor0",q
```

Comment:
did you add a "q" to the end of this line on purpose?

Reply:
Nope! Fixing